src/class-wpml-gutenberg-strings-in-block.php

```diff
@@ -25,9 +25,9 @@ public function find( WP_Block_Parser_Block $block ) {
 
 		$block_queries = $this->get_block_queries( $block );
 
-		if ( is_array( $block_queries ) ) {
+		if ( is_array( $block_queries ) && isset( $block->innerHTML ) ) {
 
-			$xpath = $this->get_domxpath( $block );
+			$xpath = $this->get_domxpath( $block->innerHTML );
```

Comment:
Not for other reviewers: this change is explained by Pierre in the comment below